### PR TITLE
uv_index, visibility, sun* added ref to #4

### PR DESCRIPTION
**Pull request by cagnulein**
Here you are! :)

### CODE REVIEW

**Comment by cagnulein** [comment]
@ErwinSteffens any news about this?

File: index.js [hunk 1]
```diff
@@ -58,6 +64,21 @@ var getForecast = function () {
...
+            var sun_status = 0;
+
+            if (generalConfig.debug) {
+                console.log("Sunrise: ", daily.sunriseTime);
+                console.log("Sunset: ", daily.sunsetTime);
+                console.log("Now: ", moment().unix());
+            }
+
+            if(moment().unix() > daily.sunriseTime && moment().unix() < daily.sunsetTime)
+                sun_status = 1;
```
Comment (ErwinSteffens) [inline index.js]: Nitpicking, but can you use the same formatting as in other places?

So camelCasing for the `sun_status` variables and braces on the correct places?

File: index.js [hunk 2]
```diff
@@ -58,6 +64,21 @@ var getForecast = function () {
...
+            var daily = forecast.daily.data[0];
+            var sun_status = 0;
+
+            if (generalConfig.debug) {
+                console.log("Sunrise: ", daily.sunriseTime);
+                console.log("Sunset: ", daily.sunsetTime);
+                console.log("Now: ", moment().unix());
+            }
+
+            if(moment().unix() > daily.sunriseTime && moment().unix() < daily.sunsetTime)
```
Comment (ErwinSteffens) [inline index.js]: Can't you just use `(new Date()).getTime() / 1000` for getting epoch in seconds? I think pulling in moment.js as a dependency just for getting the epoch time is a bit of overkill.

**Review by ErwinSteffens — CHANGES_REQUESTED** [review]
Sorry for the late reply. Thanks for the addition. 

A have a few small comments.

**Comment by cagnulein** [comment]
what do you think now?

**Comment by ErwinSteffens** [comment]
LGTM.

@cagnulein  I assume you tested it?

**Comment by cagnulein** [comment]
@ErwinSteffens yes OFC

**Comment by cagnulein** [comment]
is it ok?

**Comment by ErwinSteffens** [comment]
Yes, thnx

**Pull request merged**